tests/Doctrine/Tests/KeyValueStore/Storage/DynamoDbTest.php

```diff
@@ -310,11 +310,13 @@ public function testPassingArrayAsKeyIsAPassthruToInsert()
     public function testTryingToFindAnItemThatDoesNotExist()
     {
         $client = $this->getDynamoDbMock(['getItem']);
+        $result = $this->getDynamoDbResultMock(['get']);
```

Comment:
Could you create a new test for this, instead of changing the existing one? Or is the original test just wrong?